components/export/api/src/main/java/org/phenotips/export/internal/DataToCellConverter.java

```diff
@@ -1116,25 +1070,16 @@ public DataSection prenatalPhenotypeBody(Patient patient) throws Exception
                 cell.addStyle(lastStatus ? StyleOption.YES : StyleOption.NO);
                 section.addCell(cell);
             }
-            if (bothTypes) {
-                x++;
-            }
-            if (present.contains("name")) {
-                DataCell cell = new DataCell(feature.getName(), x, y, StyleOption.FEATURE_SEPARATOR);
-                section.addCell(cell);
-                x++;
-            }
-            if (present.contains("id")) {
-                DataCell cell = new DataCell(feature.getId(), x, y, StyleOption.FEATURE_SEPARATOR);
-                section.addCell(cell);
-                x++;
-            }
+            DataCell cell = new DataCell(feature.getName(), ++x, y, StyleOption.FEATURE_SEPARATOR);
+            section.addCell(cell);
+            section.addCell(new DataCell(feature.getId(), ++x, y, StyleOption.FEATURE_SEPARATOR));
```

Comment:
These two cells are created the same way, so why two different ways? The name cell should also be added directly, without storing it in a temporary variable.